Examples/Run/Reconstruction/RecTruthTracks.cpp

```diff
@@ -21,6 +21,7 @@
 #include "ActsExamples/Options/CommonOptions.hpp"
 #include "ActsExamples/Plugins/BField/BFieldOptions.hpp"
 #include "ActsExamples/TruthTracking/ParticleSmearing.hpp"
+#include "ActsExamples/TruthTracking/TruthSeedSelector.hpp"
```

Comment:
Did this `TruthSeedSelector` already exist?

Reply:
Yes, it's already used for CKF.